dbt/adapters/snowflake/impl.py

```diff
@@ -158,5 +160,19 @@ def quote_seed_column(self, column: str, quote_config: Optional[bool]) -> str:
         else:
             return column
 
+    @available
+    def standardize_grants_dict(self, grants_table: agate.Table) -> dict:
+        grants_dict = {}
+        
+        for row in grants_table:
+            grantee = row['grantee_name']
+            privilege = row['privilege']
+            if privilege != 'OWNERSHIP':
```

Comment:
The `OWNERSHIP` filter makes sense to me, and is one valid way of proceeding here.

My inclination would be to generalize this beyond just `OWNERSHIP`: all grants given BY the current user (=role), and NOT TO the current user (=role):
```python
for row in grants_table:

    # three relevant columns
    grantee = row['grantee_name']
    privilege = row['privilege']
    granted_by = row['granted_by']

    # super gross, but it works
    # (this is the value from profiles.yml)
    current_role = self.connections.profile.credentials.role
    
    if (
        # granted BY the current role
        granted_by.lower() == current_role.lower() and
        
        # NOT granted TO the current role
        grantee.lower() != current_role.lower()
    ):
        ...
```